app/controllers/admin/group_roles_controller.rb

```diff
@@ -59,7 +59,8 @@ def cannot_remove_admin_role_from_admin_group
         role = Role.find_by(id: params[:role_id])
         return unless @group.name == ::Ability.admin_group_name && role.name == 'admin'
 
-        redirect_back(fallback_location: edit_admin_group_path(@group), flash: { error: "Admin role cannot be removed from this group" })
+        redirect_back(fallback_location:
+          edit_admin_group_path(@group), flash: { error: "Admin role cannot be removed from this group" })
```

Comment:
This will work, but could be more readable: 

```suggestion
        redirect_back(
          fallback_location: edit_admin_group_path(@group),
          flash: { error: "Admin role cannot be removed from this group" }
        )
```

Reply:
(There are a few of this, so simply clicking `Commit suggestion` won't get all of them unfortunately)

Reply:
Agreed on this, thank you! Will make those changes

Reply:
I think I found them all and made those changes. Thank you!